### DISCUSSION TITLE
Problems with integrity checking

### DISCUSSION THREAD

**Discussion by avKostanov** [MIC-DKFZ/nnUNet#1046] [Q&A]
Hello everyone, I'm training model on CT. After verifying all the cases, it says that there are missing cases in dataset.json. But all is on it's place (labelsTr have same structure). What can be the problem& Thank you in advance
![image](https://user-images.githubusercontent.com/56444258/167253514-4e4970be-4d6d-4049-b291-88ac3ebfeb54.png)

**Comment by avKostanov** [comment]
UPDATE: even with adding modality and specifying full paths, another problem occurs
![image](https://user-images.githubusercontent.com/56444258/167254288-c7a9804f-e624-4965-80f1-33eb7eccd9cb.png)

**Comment by FabianIsensee** [comment]
It seems like some of your files are misnamed, or you placed the _0000 modality identifier in the wrong field in the dataset json. Can you please share the dataset.json?

**Comment by FabianIsensee** [comment]
Please consider using https://github.com/MIC-DKFZ/nnUNet/blob/fd58e25a304e2ab0cd4c16d2c79505c2882e3593/nnunet/dataset_conversion/utils.py#L27
